frontend/src/modules/signup/view.jsx

```diff
@@ -243,10 +254,23 @@ const SignUp = ({ match: { params }, ...props }) => {
     setFormSchema({ schema: schema, loading: true });
   }, [highlight]);
 
+  const isLoaded = () => {
+    console.log(storeData);
+    return Boolean(
+      countries.length &&
+        !isEmpty(tags) &&
+        regionOptions.length &&
+        sectorOptions.length &&
+        organisationType?.length &&
+        meaOptions?.length &&
+        stakeholders?.length
```

Comment:
I think we return different kinds of data for stakeholders if the user is admin and not admin. [I left the existing API as it is for a normal user](https://github.com/akvo/unep-gpml/blob/main/backend/src/gpml/handler/stakeholder.clj#L26), when implementing the admin response to not break any uses of the API on this branch. I think this would be a good time to make the responses look somewhat similar, if not entirely? If not, we could use the `isEmpty` util from `lodash` that we are using else where?

Reply:
using isEmpty in all conditions and unifying backend responses